src/recipes/cinnamon_maple_whiskey.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "Cinnamon Maple Whiskey Sour>",
+    "description": "Meet the best whiskey sour recipe—it’s full of bourbon and fresh lemon, sweetened with maple syrup and includes a hint of cinnamon.",
+    "github": "jjohn50",
+    "ingredients": [
+        {
+            "quantity": "3",
+            "measure": "tablespoons",
+            "ingredient": "Bulleit bourbon or your bourbon of choice"
+        },
+        {
+            "quantity": "2",
+            "measure": "tablespoons",
+            "ingredient": "Fresh lemon juice"
+        },
+        {
+            "quantity": "2-4",
+            "measure": "teaspoons",
+            "ingredient": "Maple Syrup "
+        },
+        {
+            "quantity": "1",
+            "measure": "pinch",
+            "ingredient": "Ground cinnamon"
+        }
+    ],
+    "directions": [
+        "Fill a cocktail shaker or mason jar about two-thirds full with ice. ",
+        "Pour in the bourbon, lemon juice, maple syrup and a pinch of ground cinnamon. ",
+        "Securely fasten the lid and shake well.",
+        "Fill a cocktail shaker or mason jar about two-thirds full with ice. ",
+        "Pour fresh ice into your cocktail glass and strain the cold whiskey sour mixture into the glass. Enjoy!"
+    ],
+    "image": "cinnamon-maple-whiskey-sour.jpg",
+    "source": "https://cookieandkate.com/images/2015/12/cinnamon-maple-whiskey-sour-3-768x1056.jpg",
```

Comment:
Link to the recipe here, instead of the image.

Reply:
The link is the link to the recipe from the website I found it from.

Reply:
Ok, you can remove this line if there's no source page.